specifications/active/aRMT-healthkit-1.0.0.yml

```diff
@@ -0,0 +1,82 @@
+name: aRMT
+vendor: RADAR
+model: aRMT-App
+version: 1.0.0
+assessment_type: QUESTIONNAIRE
+doc: aRMT Healthkit data definition.
+data:
+  - doc: A quantity sample type that measures the number of steps the user has taken, unit is count.
+    topic: active_healthkit_steps
```

Comment:
should we have `apple` in the topic name (likely replace `active`)?

Reply:
Should we make it `active_apple_healthkit_steps`? So we know it's from the active app?